client/state/inline-help/reducer.js

```diff
@@ -82,6 +83,11 @@ export const search = withoutPersistence(
 					selectedResult: -1,
 					hasAPIResults: false,
 				};
+			case INLINE_HELP_SET_SEARCH_QUERY:
+				return {
+					...state,
+					searchQuery: action.searchQuery,
+				};
 			case INLINE_HELP_SEARCH_REQUEST:
```

Comment:
WE leave this for a future follow up, but with `INLINE_HELP_SET_SEARCH_QUERY` we can replace instances of `INLINE_HELP_SEARCH_REQUEST` as this appears to do the same thing and setting a query is more descriptive to the app state here.

Reply:
sounds good to me https://github.com/Automattic/wp-calypso/issues/43845